server/utils/bot/ensure.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package bot
+
+import (
+	"io/ioutil"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+	"github.com/pkg/errors"
+)
+
+func EnsureWithProfileImage(api plugin.API, h plugin.Helpers, username, displayName, description, imagePath string) (string, error) {
```

Comment:
Separate HW ticket? @crspeller suggested that some of these should be moved to p.Helpers, eventually - maybe combine?

Reply:
FYI: Since 5.18 `EnsureBot` support setting the profile image. See https://developers.mattermost.com/extend/plugins/server/reference/#Helpers.EnsureBot

Reply:
I'll use that